ui/apps/portal/src/plugins/Setup/Db/SaveDbDialog/SaveDbDialog.tsx

```diff
@@ -194,6 +194,9 @@ export const SaveDbDialog: FC<SaveDbDialogProps> = ({ open, onClose }) => {
         formData.port = 0;
       }
 
+      if(!isValidDbCode(formData.code, setFeedback)) {
+        return;
```

Comment:
instead of just returning, can use `setFeedback` with an error message, to inform the user what should be the valid db code